inst/shiny/DiagnosticsExplorer/global.R

```diff
@@ -1,158 +1,315 @@

-    )

-} else {

-  # For backwards compatibility with older versions of DatabaseConnector:

-  connectionPool <-

-    pool::dbPool(

-      drv = DatabaseConnector::DatabaseConnectorDriver(),

-      dbms = connectionDetails$dbms,

-      server = connectionDetails$server,

-      port = connectionDetails$port,

-      user = connectionDetails$user,

-      password = connectionDetails$password,

-      connectionString = connectionDetails$connectionString

-    )

-}

-

-dataModelSpecifications <-

-  read.csv("resultsDataModelSpecification.csv")

-# Cleaning up any tables in memory:

-suppressWarnings(rm(

-  list = SqlRender::snakeCaseToCamelCase(dataModelSpecifications$tableName)

-))

-

-

-onStop(function() {

-  if (DBI::dbIsValid(connectionPool)) {

-    writeLines("Closing database pool")

-    pool::poolClose(connectionPool)

-  }

-})

-

-resultsTablesOnServer <-

-  tolower(DatabaseConnector::dbListTables(connectionPool, schema = resultsDatabaseSchema))

-

-loadResultsTable("database", required = TRUE)

-loadResultsTable("cohort", required = TRUE)

-loadResultsTable("temporal_time_ref")

-loadResultsTable("concept_sets")

-loadResultsTable("cohort_count", required = TRUE)

-

-for (table in c(dataModelSpecifications$tableName)) {

-  #, "recommender_set"

-  if (table %in% resultsTablesOnServer &&

-      !exists(SqlRender::snakeCaseToCamelCase(table)) &&

-      !isEmpty(table)) {

-    #if table is empty, nothing is returned because type instability concerns.

-    assign(SqlRender::snakeCaseToCamelCase(table),

-           dplyr::tibble())

-  }

-}

-

-dataSource <-

-  createDatabaseDataSource(

-    connection = connectionPool,

-    resultsDatabaseSchema = resultsDatabaseSchema,

-    vocabularyDatabaseSchema = resultsDatabaseSchema

-  )

-

-

-if (exists("database")) {

-  if (nrow(database) > 0 &&

-      "vocabularyVersion" %in% colnames(database)) {

-    database <- database %>%

-      dplyr::mutate(

-        databaseIdWithVocabularyVersion = paste0(databaseId, " (", .data$vocabularyVersion, ")")

-      )

-  }

-}

-

-if (exists("cohort")) {

-  cohort <- get("cohort")

-  cohort <- cohort %>%

-    dplyr::arrange(.data$cohortId) %>%

-    dplyr::mutate(shortName = paste0("C", dplyr::row_number())) %>%

-    dplyr::mutate(compoundName = paste0(.data$shortName, ": ", .data$cohortName,"(", .data$cohortId, ")"))

-}

-

-if (exists("temporalTimeRef")) {

-  temporalCovariateChoices <- get("temporalTimeRef") %>%

-    dplyr::mutate(choices = paste0("Start ", .data$startDay, " to end ", .data$endDay)) %>%

-    dplyr::select(.data$timeId, .data$choices) %>%

-    dplyr::arrange(.data$timeId)

-}

-

-if (exists("covariateRef")) {

-  specifications <- readr::read_csv(

-    file = "Table1Specs.csv",

-    col_types = readr::cols(),

-    guess_max = min(1e7)

-  )

-  prettyAnalysisIds <- specifications$analysisId

-} else {

-  prettyAnalysisIds <- c(0)

-}

-
+library(magrittr)
```

Comment:
what happened here

Reply:
It looks like In a previous commit windows line endings got inserted (the ^M character)  I've seen this happen before with different IDEs, but this got interpreted as an extra line ending and hence why it was messed up. It should be fixed now.